### PR TITLE
making the external assets form optional

### PR DESCRIPTION
**Pull request by jcamejo**
Hi @artf i hope you are having a nice day.

What do you think of this small improvement? I just got a use case where i needed to have this option and i think is a good idea.

Thanks!

### CODE REVIEW

**Comment by artf** [comment]
Yeah cool, seems like a good idea 👍, but can you check the failing tests, please?

**Comment by jcamejo** [comment]
Sorry for leaving that behind, the tests are added. There are some redeclarations of the test object to make clear that the view is being initialized with the correct config

File: src/asset_manager/view/AssetsView.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default Backbone.View.extend({
     const url = input.value.trim();
     const handleAdd = this.config.handleAdd;
 
-    if (!url) {
+    if (!url || !input) {
```
Comment (artf) [inline src/asset_manager/view/AssetsView.js]: mmm this `!input` check doesn't make sense here, it that case it would crash at `input.value.trim()`

**Comment by jcamejo** [comment]
Absolutely right, it's changed.

**Comment by jcamejo** [comment]
I'm not sure if this failure has something to do with the change, it does not look like it. Should i check some other module that i'm not aware of?

Thanks again for your assistance.

**Comment by artf** [comment]
> I'm not sure if this failure has something to do with the change, it does not look like it

Nope, my fault 😬

Thanks for the contribution Juan

**Pull request merged**